### PR TITLE
New structure and docs

### PR DESCRIPTION
**Pull request by alexmuller**
I've been thinking a bit about the things I've learnt in the last couple of weeks and what might be useful for the organisation to write down.

I think it would be really helpful to have a brief overview of all of the different projects that are going on, where they're hosted and how they interact with each other.

This will make onboarding much easier and will hopefully give everyone a much better idea of what's going on.

Let me know if:

- it's useful
- it's too much
- it already exists and I haven't found it

### CODE REVIEW

File: docs/projects/tuneme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Tune Me
+
+[Tune Me](https://tuneme.org/) is a mobile site for providing health
+information to people in Zambia.
```
Comment (JayH5) [inline docs/projects/tuneme.md]: Not just Zambia afaik.. 5 countries I think. Best to ask Katlego.

File: docs/tech/logging.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Logging
+
+All logging must happen to a standardized path, under `/var/praekelt/logs`. If your app
+has a large amount of files (celery, worker, and app, for instance), write each of
+these to a named path under `/var/praekelt/logs/appname/foo.log`, otherwise just
+`/var/praekelt/logs/foo.log` is sufficient.
+
+Logs always end with the extension `.log`. `.err` is not valid. If you need to write out
+error logs, either use the name `foo-error.log`, or write your supervisord configuration
+with the `redirect_stderr` option.
```
Comment (JayH5) [inline docs/tech/logging.md]: Is this being copy/pasted from the existing docs somewhere? These are mostly conventions from Sideloader days and not relevant with containers on the cluster.

File: docs/tech/python-packages.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Python packages
+
+A lot of our code is published as Python packages on [PyPI][].
+
+[PyPI]: https://pypi.org/
+
+## Development
+
+Make changes to the repository that contains the Python package using
+the normal GitHub flow.
```
Comment (JayH5) [inline docs/tech/python-packages.md:10]: This is slightly confusing because there is also git flow, the tool. The tagging and version bump below can be done using `git flow release` which is probably what we would recommend in most cases (I think).

File: docs/tech/vumi.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+We build on top of our Vumi platform for SMS, USSD and other messaging protocols.
+
+[Vumi][] is a scalable messaging engine which we use for SMS, USSD and other messaging
+protocols.
+
+We [build Vumi in a container][docker-vumi] and [push it to Docker Hub][dockerhub].
+
+[Vumi Go][] is a hosted version of Vumi. Where Vumi gives you the tools to
+build large scale messaging applications, Vumi Go provides you with a working
+environment that is already integrated into numerous countries.
```
Comment (JayH5) [inline docs/tech/vumi.md]: Not sure if you want to mention that Vumi Go is deprecated and going away by the end of the year.

**Review by JayH5 — COMMENTED** [review]
🍰 🍰 🍰

File: docs/tech/logging.md (hunk 2 above)
Comment (alexmuller) [reply docs/tech/logging.md]: Yeah, this is from https://ways-of-working.readthedocs.io/en/latest/platforms.html#logging - I removed a lot of the stuff about how we deploy Django (because I assume that's also containers) but I didn't know if the logging was still relevant. Will remove!

**Comment by alexmuller** [comment]
Thanks for reviewing this @JayH5, I think I've addressed your comments! :)

File: docs/tech/python-packages.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Bump the version in the repository. You might have to look in:
+
+- `setup.py`
+- `package/__init__.py`
+- `docs/conf.py`
+
+## Release
+
+Use [`git flow release`](../tools.html#git-flow) to publish a new release
+to GitHub. This will create a new tag which will cause Travis to build.
```
Comment (JayH5) [inline docs/tech/python-packages.md:21]: There is probably more we could write here... `develop` branch is usually the default branch, PRs target that branch, `master` tracks the latest releases, etc. But maybe that should be in a different PR? Or maybe people should just read on the git flow page? Not sure how obvious this is.

Also, `git flow release` doesn't push anything, you still have to push the branches and tag yourself.

File: docs/tech/django.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Django
+
+We use the [Django][] Python framework heavily.
+
+You should use the latest stable release of Django *as of the start of the project*
+unless otherwise specified. Pin that version in your requirements so that the
+project won't break by accidentally being deployed on a newer version without
+testing.
+
+We strongly recommend you use our sample [django-skeleton][] as a starting point for
```
Comment (JayH5) [inline docs/tech/django.md]: The `django-skeleton` project is _very_ out-of-date and I'd prefer we didn't recommend it until/unless someone significantly updates it 😿

File: docs/tech/junebug.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Junebug
+
+[Junebug][] is an HTTP API for sending messages. It uses code from [vumi][].
```
Comment (JayH5) [inline docs/tech/junebug.md]: "sending messages" is not really the way I've thought about Junebug, but maybe that is the simplest way to put it. The intro sentence on the Junebug readthedocs is... not really easy to understand at all 😓 .

Instead of saying _"It uses code from Vumi"_ you could rather say something like "It provides an API for configuring Vumi transports"... but I'm not sure that's any more clear?

File: docs/tech/python-packages.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+The `.travis.yml` file will deploy the package to PyPI as the
+[`praekelt.org` user](https://pypi.org/user/praekelt.org/).
+
+For each package repository, there's another repo prefixed with `docker-`. For
```
Comment (JayH5) [inline docs/tech/python-packages.md]: "For many package repositories" maybe?

File: docs/tech/vumi.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# Vumi
+
+We build on top of our Vumi platform for SMS, USSD and other messaging protocols.
+
+[Vumi][] is a scalable messaging engine which we use for SMS, USSD and other messaging
+protocols.
+
+We [build Vumi in a container][docker-vumi] and [push it to Docker Hub][dockerhub].
```
Comment (JayH5) [inline docs/tech/vumi.md:8]: Could maybe add something like: "Nowadays, for the most part Vumi is only used via Junebug." Except my English is bad right now.

**Review by JayH5 — COMMENTED** [review]
Some nitpicking mostly. I don't really want us to recommend `django-skeleton`, though.

File: docs/tech/python-packages.md (hunk 5 above)
Comment (alexmuller) [reply docs/tech/python-packages.md:21]: I don't think I know enough about this to write the docs on it yet, but either somebody else can or I will do once I've figured out the details :)

**Comment by alexmuller** [comment]
Thanks again, updated to clarify things.

File: docs/tech/django.md (hunk 6 above)
Comment (alexmuller) [reply docs/tech/django.md]: Agreed, I was concerned by the date of the last commit!

**Review by JayH5 — APPROVED** [review]
:+1: from me
